

# User not found exception
class UserNotFound(Exception):
    pass